Sources/OAuthSwiftHTTPRequest.swift

```diff
@@ -428,16 +447,13 @@ public struct URLSessionFactory {
     /// An optional delegate for the URLSession
     public weak var delegate: URLSessionDelegate?
 
-    /// Monitor session: see UIApplication.shared.isNetworkActivityIndicatorVisible
-    public var isNetworkActivityIndicatorVisible = true
-
     /// By default use a closure to receive data from server.
     /// If you set to false, you must in `delegate` take care of server response.
     /// and maybe call in delegate `OAuthSwiftHTTPRequest.completionHandler`
     public var useDataTaskClosure = true
```

Comment:
I was going to complain about this name and the comment above but I noticed that it's already been there. 😉